services/jitpack/jitpack-version-redirector.service.js

```diff
@@ -5,10 +5,20 @@ export default [
     category: 'version',
     route: {
       base: 'jitpack/v',
-      pattern: ':groupId/:artifactId',
+      pattern: ':user/:repo',
     },
-    transformPath: ({ groupId, artifactId }) =>
-      `/jitpack/v/github/${groupId}/${artifactId}`,
+    transformPath: ({ user, repo }) =>
+      `/jitpack/version/com.github.${user}/${repo}`,
     dateAdded: new Date('2019-03-31'),
   }),
+  redirector({
```

Comment:
The reason the tests are failing is:

```
Error: Duplicate service names found: {
  "JitpackVRedirect": 2
}
```

we'll need to manually give each of these redirects a unique name when we declare them as the auto-generated names are clashing.

You can run `npm test` locally to run the core tests and liners

Reply:
Thank you; I couldn't get `npm install` to work... I've fixed it and was finally able to test the code.